examples/matRad_example16_lexicographicOptimization.m

```diff
@@ -0,0 +1,51 @@
+matRad_rc; %If this throws an error, run it from the parent directory first to set the paths
```

Comment:
Same as for example15

Reply:
Have to check this one, but I think the same issue with the results not making sense at all willl show up.